docs/_includes/errormessages.adoc

```diff
@@ -0,0 +1,274 @@
+
+[appendix]
+== Error Messages
+
+// use <source> for an asciidoc source file with line number
+// use <file> for an arbitrary file
+// use <x,y,z> for anything else
+
+////
+Testing was carried out with
+Asciidoctor 1.5.5 [http://asciidoctor.org]
+Runtime Environment (ruby 2.3.1p112 (2016-04-26) [i386-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:- ex:UTF-8)
+////
+
+The most common error and warning messages when using the asciidoctor command line (including asciidoctor-ant) are listed alphabetically in the table below.
+Messages from asciidoctorj or the asciidoctor API are not included.
```

Comment:
Add a comment to this line about asciidoctorj and asciidoctor API. I think that just confuses the reader for now, but it is important information for the documenter.